### DISCUSSION TITLE
Entangle with Alpine 3 not working

### DISCUSSION THREAD

**Discussion by arnaudlidz** [livewire/livewire#3188] [7. Bugs (archived)]
Hello,

Entangle not working properly with Alpine 3. It's actually get value on init but no more ajax call are made to update the value.
Use last 2.X version solve the problem instantly

Best regards

**Comment by joshhanley** [comment]
@arnaudlidz can you please share some code, so we can try and replicate the issue? I've tested `@entangle` and haven't had any issues with it.

Thanks!

**Reply by arnaudlidz** [reply]
In fact I have this error : "Uncaught (in promise) TypeError: window.Alpine.disableEffectScheduling is not a function" and It seems that Livewire loose alpine root properties when refresh. For the code, the simple dropdown example in the livewire docs here doesn't work.

I gone deeper and I found a conflict with my app.js and alpinejs v3. However this is the default start app.js file. I commented the require alpine line but not working. I really don't know where this bug come from. Last Laravel, Livewire, Alpine version.

**Reply by joshhanley** [reply]
@arnaudlidz can you share your app.js file where you import and start Alpine? There was a change in V3 on how to import and start Alpine, see here https://alpinejs.dev/upgrade-guide#need-to-call-alpine-start

Also make sure you include you app script tag after Livewire scripts or add a `defer` to it, if it's in your `<head>` tag.

Hope this helps!

**Reply by arnaudlidz** [reply]
Thank you @joshhanley , my app.js contain only the require bootstrap line and by default use defer to import it in header, import in footer after livewire did not fix the problem. I import Alpine with unpkg cdn url, so I really don't know. Because I don't need app.js file, it's work but it isn't a long term solution.

**Reply by joshhanley** [reply]
@arnaudlidz ah right, didn't realise you were also using CDN, I was confused by the "I commented the require alpine line".

Yeah if you want to use your app.js file instead, I would consider removing the CDN and using NPM to install Alpine V3 and use the new method in the docs to import and start Alpine.

**Reply by calebporzio** [reply]
Alpine V3 exposes that method on the Alpine global: 
![image](https://user-images.githubusercontent.com/3670578/123652252-1fc91600-d7fa-11eb-9b1a-a82527847bdd.png)

If you are getting that message, it tells me that you are either on the wrong version of Alpine or something deeper is happening (the former seems more likely to me)
